rsconf/component/elasticsearch.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+u"""create elasticsearch configuration
+:copyright: NA
+:license: NA
+"""
+import os
+from __future__ import absolute_import, division, print_function
+from pykern.pkcollections import PKDict
```

Comment:
Remove `__future__`, and from the module you copied it from. Add 

from pykern.pkdebug import pkdp